Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,64 @@
+var botOptions = ["rock", "paper", "scissors"];
+
+var humanScore = 0;
+
+var computerScore = 0;
+
+function humanRock (e) {
+  var humanChoice = $("#rock").attr("id");
+  var botChoice = botOptions[Math.floor(Math.random() * botOptions.length)];
+  console.log(botChoice);
+  compare(humanChoice, botChoice);
+}
+
+function humanPaper (e) {
+  var humanChoice = $("#paper").attr("id");
+  var botChoice = botOptions[Math.floor(Math.random() * botOptions.length)];
+  console.log(botChoice);
+  compare(humanChoice, botChoice);
+}
+
+function humanScissors (e) {
```

Comment:
As you're not doing anything with the event (my guess is you were console.log'ing it when you first set the three wrapper functions up, you might consider removing the reference to the event argument, `e`